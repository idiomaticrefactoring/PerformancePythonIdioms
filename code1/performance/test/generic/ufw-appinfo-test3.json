[{"profile":"TEST3","title":"My test app3","description":"test overlapping ports","tcp_list":[80,83,80,53],"tcp_ranges":[{"start":70,"end":90}],"udp_ranges":[{"start":50,"end":51}],"udp_list":[53],"normalized_tcp_list":[53],"normalized_tcp_ranges":[{"start":70,"end":90}],"normalized_udp_list":[53],"normalized_udp_ranges":[{"start":50,"end":51}]}]
