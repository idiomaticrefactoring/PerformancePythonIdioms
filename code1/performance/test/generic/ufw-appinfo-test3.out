Profile: TEST3
Title: My test app3
Description: test overlapping ports

Ports:
  80,83,80,70:90/tcp
  50:51/udp
  53

